dist/images/daemonset.sh

```diff
@@ -858,6 +880,8 @@ ovn_image=${ovnkube_image} \
   ovn_enable_interconnect=${ovn_enable_interconnect} \
   ovn_enable_multi_external_gateway=${ovn_enable_multi_external_gateway} \
   ovn_enable_ovnkube_identity=${ovn_enable_ovnkube_identity} \
+  ovn_transit_switch_subnet_ipv4=${ovn_transit_switch_subnet_ipv4} \
```

Comment:
does ovnkube-zone-controller in IC need this?

Reply:
ovnkube-controller container of ovnkube-zone-controller.yaml runs ovnkube-controller() in ovnkube.sh. ovnkube-controller() sets --init-ovnkube-controller while starting OVN. As per my understanding transit switch subnet is required while running with --init-ovnkube-controller flag and interconnect is enabled.

~~~
command: ["/root/ovnkube.sh", "ovnkube-controller"]
~~~

Reply:
so isn't the transit switch subnet actually given to each zone by the cluster-manager? isn't that why its called ClusterManager config? Where in zone controller do we use the transit switch config in?

Reply:
yes, cluster-manager assigns transit switch subnet and adds node-transit-switch-port-ifaddr annotation. network-controller-manager retrieves transit_switch subnet from annotation and configures logical router/switch.  
I will have this rectified in next push.